src/components/Table/Table.js

```diff
@@ -111,7 +111,7 @@ class Table extends PureComponent {
 
   selectRow({ index }) {
     const { expandedHeight } = this.props;
-    const { selectedIndex } = this.state;
+    const { selectedIndex, hoverIndex } = this.state;
```

Comment:
do we need `hoverindex` here? It's not used in this function

Reply:
Ah, no. Doesn't look like it. Wonder if this got left from something else I was trying 🤔. Linter should have caught this too. This is why we need CI! (though I guess it wouldn't have mattered for our weird incestuous PR branch on a PR branch 😬)